SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -754,14 +754,31 @@ bool FSpatialGDKEditorToolbarModule::FetchInspectorBinaryWrapper(FString Inspect
 {
 	bFetchingInspectorBinary = true;
 
-	const bool bSuccess =
-		SpatialCommandUtils::FetchInspectorBinary(InspectorVersion, GetDefault<USpatialGDKSettings>()->IsRunningInChina());
+	bool bSuccess = SpatialCommandUtils::FetchInspectorBinary(InspectorVersion, GetDefault<USpatialGDKSettings>()->IsRunningInChina());
 
 	if (!bSuccess)
 	{
 		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("Could not fetch the Inspector for version %s"), *InspectorVersion);
 		OnShowFailedNotification(TEXT("Failed to fetch local inspector!"));
+
+		bFetchingInspectorBinary = false;
+		return false;
+	}
+
+#if PLATFORM_MAC
+		int32 OutCode = 0;
```

Comment:
Huh, I thought linter would pick this up.
```suggestion
	int32 OutCode = 0;
```

Reply:
ah could be that either  I didn't check it after linting or it didn't work properly on mac. we really should add linting to ci